.gitignore

```diff
@@ -2,3 +2,4 @@
 *.pyc
 __pycache__
 .idea/
+venv
```

Comment:
I believe this is virtual environment name, and since it could be different, I would suggest to remove from the root project `.gitignore` file.

Reply:
Done!